sway/tree/container.c

```diff
@@ -573,16 +573,22 @@ void container_update_title_textures(struct sway_container *container) {
 }
 
 void container_calculate_title_height(struct sway_container *container) {
+        int height;
```

Comment:
With project code style, these lines should be moved after the `if (!container->formatted_title)` block.